uber/utils.py

```diff
@@ -232,11 +264,28 @@ def genpasswd():
     except:
         return ''.join(chr(randrange(33, 127)) for i in range(8))
 
-@entry_point
-def print_config():
+
```

Comment:
I think the docstrings sufficiently explain this, but let me know if they don't.